src/main/java/emissary/place/ServiceProviderRefreshablePlace.java

```diff
@@ -78,21 +90,49 @@ public ServiceProviderRefreshablePlace(final ServiceProviderRefreshablePlace pla
         setupPlace(null, place.placeLocation, register);
     }
 
+    @Override
+    protected void setupPlace(@Nullable final String theDir, final String placeLocation, final boolean register) throws IOException {
+        super.setupPlace(theDir, placeLocation, register);
+
+        try {
+            Preconditions.checkNotNull(this.configG, "The configurator is null");
+            final var path = configG.findStringEntry("MONITORING_PATH");
+            if (StringUtils.isNotBlank(path)) {
```

Comment:
Should we also add a `Files.exists(file)` check here or in the `Monitor` constructor?